### DISCUSSION TITLE
numpy.core._exceptions._ArrayMemoryError: Unable to allocate XXMiB for an array with shape (XXX,)

### DISCUSSION THREAD

**Discussion by nikhartz** [danielhrisca/asammdf#971] [Q&A]
Hi,
i am using the asammdf tool and everytime i want to convert a .mf4-file into an .parquet-file, i am getting the Error in the title.
The option to run it with less ram also doesn't make any difference. My .mf4 is about 100mb big.

Is this problem known and does a fix exist for that ?

**Comment by danielhrisca** [comment]
Please send your file for analysis
